docs/modules/ROOT/pages/udc.adoc

```diff
@@ -1,17 +1,12 @@
 = Universal Deployer Contract
 
-Account contracts make up very critical components of the StarkNet ecosystem, since a bug in any implementation (let alone a widespread one) could prove disastrous.
-Therefore, maximum caution is in order.
-In this spirit, we should consider trimming account responsibilities to simplify implementations and minimize bug/attack surfaces.
+The Universal Deployer Contract (UDC) is a singleton smart contract that wraps the https://www.cairo-lang.org/docs/hello_starknet/deploying_from_contracts.html#the-deploy-system-call[`deploy`] syscall.
+This contract intends to serve and support all other contracts that do not implement this functionality (like most accounts).
+And since accounts lack a native deployment transaction type, the UDC aims to stand in this while also providing a standardized event to ease observability.
```

Comment:
```suggestion
The Universal Deployer Contract (UDC) is a singleton smart contract that wraps the https://www.cairo-lang.org/docs/hello_starknet/deploying_from_contracts.html#the-deploy-system-call[`deploy syscall`] to expose it to any contract that doesn't implement it, such as account contracts. You can think of it as a **standardized generic factory for StarkNet contracts**.

And since StarkNet has no deployment transaction type, it offers a canonical way to deploy smart contracts by following the [standard deployer interface](https://community.starknet.io/t/snip-deployer-contract-interface/2772) and emitting a `ContractDeployed` event.
``````
how does this read?

Reply:
Much better!